src/ui/components/shared/TeamOnboarding.tsx

```diff
@@ -73,11 +74,11 @@ function TeamNamePage({
   const createNewWorkspace = hooks.useCreateNewWorkspace(onNewWorkspaceCompleted);
   const updateDefaultWorkspace = hooks.useUpdateDefaultWorkspace();
 
-  function onNewWorkspaceCompleted(workspace: Workspace) {
+  function onNewWorkspaceCompleted(workspace: CreateNewWorkspace_createWorkspace_workspace | null) {
```

Comment:
> Is this a case of a valid type but invalid semantics?

That's what it seems to me. I've added a corresponding `assert` to `useCreateNewWorkspace()` so that I could make the type here non-nullable.